index.html

```diff
@@ -0,0 +1,874 @@
+<!doctype html>
+<html lang="zxx">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.css">
+    <title>Newsweek clone</title>
+  </head>
+  <body>
+    <!-- Header of the Page. It contains name of the site and a navbar -->
+    <!-- Navbar transforms the header in a navbar -->
+    <nav class="navbar navbar-expand-lg sticky-top flex-column navbar-dark">
+      <!-- Red part of navbar -->
+      <div class="d-flex flex-row align-self-start justify-content-start navcolor w-100 px-2 justify-content-lg-between">
+        <a class="navbar-brand order-lg-2" href="#"> <img src="img/Newsweek_Logo.png" alt="Newsweek Logo"> </a>
+        <div class="weather-time align-self-center mr-auto order-lg-1 d-lg-flex flex-column mr-lg-0 ml-lg-2">
+          <a href="#"><img src="https://g.newsweek.com/img/weather/34.png" alt="weather">18° <span class="country">Tlajomulco, Mx ></span></a>
+          <time class="d-none d-lg-block" datetime="2020-03-20">Fri, Mar 20, 2020</time>
+        </div>
+        <div class="align-self-end d-block m-3 order-lg-3 h5">
+          <span class="text-uppercase font-weight-bold text-white px-2"><a href="#">Sign in</a></span>
+          <span class="text-uppercase font-weight-bold bg-white px-2 py-1"><a href="#" id="sign">Subscribe ></a></span>
+        </div>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+        </button>
+      </div>
```

Comment:
- [ ] Your navbar has more height than the original one. Please make the adjustment.